upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -728,24 +728,34 @@ func (c *ApplyClusterCmd) Run() error {
 	}
 	switch kops.CloudProviderID(cluster.Spec.CloudProvider) {
 	case kops.CloudProviderAWS:
-		awsModelContext := &awsmodel.AWSModelContext{
-			KopsModelContext: modelContext,
-		}
+		{
+			awsModelContext := &awsmodel.AWSModelContext{
+				KopsModelContext: modelContext,
+			}
 
-		if featureflag.Spotinst.Enabled() {
-			l.Builders = append(l.Builders, &spotinstmodel.InstanceGroupModelBuilder{
+			awsModelBuilder := &awsmodel.AutoscalingGroupModelBuilder{
 				AWSModelContext:   awsModelContext,
 				BootstrapScript:   bootstrapScriptBuilder,
 				Lifecycle:         &clusterLifecycle,
 				SecurityLifecycle: &securityLifecycle,
-			})
-		} else {
-			l.Builders = append(l.Builders, &awsmodel.AutoscalingGroupModelBuilder{
-				AWSModelContext:   awsModelContext,
+			}
+
+			spotinstModelBuilder := &spotinstmodel.InstanceGroupModelBuilder{
```

Comment:
Moving this outside the featureflag _looks_ scary, but it's fine :-)

Reply:
Got it, [fixed](https://github.com/kubernetes/kops/blob/163f878ee0653144d71684cfa564bf492071ba7a/upup/pkg/fi/cloudup/apply_cluster.go#L744-L749).